UI/window-basic-status-bar.hpp

```diff
@@ -3,30 +3,40 @@
 #include <QStatusBar>
 #include <QPointer>
 #include <QTimer>
-#include <util/platform.h>
 #include <obs.h>
+#include <memory>
 
+class Ui_StatusBarWidget;
 class QLabel;
 
+class StatusBarWidget : public QWidget {
+	Q_OBJECT
+
+	friend class OBSBasicStatusBar;
+
+private:
+	std::unique_ptr<Ui_StatusBarWidget> ui;
+
+public:
+	StatusBarWidget(QWidget *parent = nullptr);
+	~StatusBarWidget();
```

Comment:
```suggestion
	inline ~StatusBarWidget(){};
```

Reply:
The destructor needs to be in the cpp file for the std::unique_ptr to work.